ci_tools/bot_tools/bot_funcs.py

```diff
@@ -418,22 +418,24 @@ def is_test_required(self, commit_log, name, key, state):
         """
         print("Checking : ", name)
         if key in ('linux', 'windows', 'macosx', 'anaconda_linux', 'anaconda_windows', 'coverage', 'intel'):
-            has_relevant_change = lambda diff: any((f.startswith('pyccel/') or f.startswith('tests/')) \
-                                                    and f.endswith('.py') and f != 'pyccel/version.py' \
-                                                    for f in diff) #pylint: disable=unnecessary-lambda-assignment
+            has_relevant_change = lambda diff: any((f.startswith('pyccel/') or f.startswith('tests/'))  #pylint: disable=unnecessary-lambda-assignment
+                                                    and f.endswith('.py') and f != 'pyccel/version.py'
+                                                    for f in diff)
         elif key in ('pyccel_lint'):
-            has_relevant_change = lambda diff: any(f.startswith('pyccel/') and f.endswith('.py') \
-                                                    and f != 'pyccel/version.py' for f in diff) #pylint: disable=unnecessary-lambda-assignment
+            has_relevant_change = lambda diff: any(f.startswith('pyccel/') and f.endswith('.py') #pylint: disable=unnecessary-lambda-assignment
+                                                    and f != 'pyccel/version.py' for f in diff)
         elif key in ('pylint'):
             has_relevant_change = lambda diff: any(f == 'pyccel/parser/semantic.py' for f in diff) #pylint: disable=unnecessary-lambda-assignment
         elif key in ('docs'):
-            has_relevant_change = lambda diff: any(f.endswith('.py') and f != 'pyccel/version.py' \
-                                                    for f in diff) #pylint: disable=unnecessary-lambda-assignment
+            has_relevant_change = lambda diff: any(f.endswith('.py') and f != 'pyccel/version.py' #pylint: disable=unnecessary-lambda-assignment
+                                                    for f in diff)
         elif key in ('spelling'):
             has_relevant_change = lambda diff: any(f.endswith('.md') or f == '.dict_custom.txt' for f in diff) #pylint: disable=unnecessary-lambda-assignment
         elif key in ('pickle', 'pickle_wheel', 'editable_pickle'):
-            has_relevant_change = lambda diff: any(f.startswith('pyccel/') and f.endswith('.py') \
-                                                    and f != 'pyccel/version.py' for f in diff) #pylint: disable=unnecessary-lambda-assignment
+            has_relevant_change = lambda diff: any((f.startswith('pyccel/') and f.endswith('.py') #pylint: disable=unnecessary-lambda-assignment
+                                                    and f != 'pyccel/version.py') or f == 'pyproject.toml'
+                                                    or f == 'MANIFEST.in' or f.startswith('install_scripts/')
+                                                    for f in diff)
```

Comment:
Have you not removed `MANIFEST.in`?

Reply:
Fixed [b879e5c](https://github.com/pyccel/pyccel/pull/1638/commits/b879e5ce8444599d228ffb11dbc64fe71e4a47d5)